### PR TITLE
[zh-cn] Reword in rbac.md

### PR DESCRIPTION
**Pull request by yanrongshi**
```yaml
content\zh-cn\docs\reference\access-authn-authz\rbac.md
```

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Because ClusterRoles are cluster-scoped, you can also use them to grant access t
 -->
 ###  ClusterRole 示例 {#clusterrole-example}
 
-ClusterRole 可以和 Role 相同完成授权。
+ClusterRole 可以用于授予与 Role 相同的权限。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/rbac.md:178]:
```suggestion
ClusterRole 类似 Role，同样可用于完成相同的授权。
```
Though the original is not fluent, the meaning is correct. My suggestion is only a suggestion. Need think it over.

Comment (yanrongshi) [reply content/zh-cn/docs/reference/access-authn-authz/rbac.md:178]: > Though the original is not fluent, the meaning is correct. My suggestion is only a suggestion. Need think it over.

在 Kubernetes 种 ClusterRole 可以授予 Role 能够授予的权限，也能授予 Role 不能授予的权限，我觉得上句话更多的语义可能是表达 ` ClusterRole 同样可以授予 Role 能够授予的权限` ，您觉得呢？大佬

Comment (windsonsea) [reply content/zh-cn/docs/reference/access-authn-authz/rbac.md:178]: 😭 My typing means your change is similar to the original text. Both are not fluent, so thinking over is required 🤣

**Review by Sea-n — COMMENTED** [review]
Please avoid using `Sync file.md` if you are not sync from upstream (English). Maybe it will be better:
/retitle [zh-cn] Reword in rbac.md

Overall looks good to me.
/approve
/approve

**Pull request merged**